fe/fe-core/src/main/java/org/apache/doris/planner/PlanNode.java

```diff
@@ -175,26 +176,32 @@ protected PlanNode(PlanNodeId id, PlanNode node, String planNodeName) {
                 "V" + planNodeName : planNodeName;
         this.numInstances = 1;
         this.nodeType = node.getNodeType();
```

Comment:
```suggestion
        this.nodeType = nodeType;
```

Reply:
In order to reduce file modifications, this PR only makes special modifications to SCAN_NODE, and will subsequently initialize Node as a parameter